echopype/convert/set_groups_ad2cp.py

```diff
@@ -17,20 +20,12 @@
 class SetGroupsAd2cp(SetGroupsBase):
     """Class for saving groups to netcdf or zarr from Ad2cp data files."""
 
-    beamgroups_possible = [
-        {
-            "name": "Beam_group1",
-            "descr": (
-                "contains velocity, correlation, and backscatter power (uncalibrated)"
-                " data and other data derived from acoustic data."
-            ),
-        }
-    ]
-
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.pulse_compressed = self.parser_obj.get_pulse_compressed()
         self._make_time_coords()
+        with resources.open_text(convert, "ad2cp_fields.yaml") as f:
+            self.field_attrs: Dict[str, Dict[str, Dict[str, str]]] = yaml.safe_load(f)  # type: ignore # noqa
```

Comment:
I noticed that all the `comment` is missing a "." at the end before the sentence "Data was converted to the specified units." Could you put it in? Thanks!